### PR TITLE
SEO-172146 - Presentation duplicate description

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @Aishwarya-Ganesan,
Kindly review the changes for duplicate description.

Regards,
Sharly A.

### CODE REVIEW

File: File-Formats/Presentation/create-read-edit-powerpoint-files-in-asp-net-core-c-sharp.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Create and edit PowerPoint files in ASP.NET Core |Syncfusion|
-description: A .NET Core PowerPoint library to create, read and edit PowerPoint files in .NET Core applications. Supports text, shape, chart, table and combine PowerPoints.
+title: Create and edit PowerPoint files in ASP.NET Core | Syncfusion
+description: Learn here all about create,read and edit support in Syncfusion ASP.NET Core PowerPoint control, its features, and more.
```
Comment (gowthamanthavasiyappan) [inline File-Formats/Presentation/create-read-edit-powerpoint-files-in-asp-net-core-c-sharp.md:3]: Revert this description.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: File-Formats/Presentation/create-read-edit-powerpoint-files-in-asp-net-core-c-sharp.md (hunk 1 above)
Comment (Sharly-Atieno) [reply File-Formats/Presentation/create-read-edit-powerpoint-files-in-asp-net-core-c-sharp.md]: reverted

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**